Beeper/barcelona-mautrix/BarcelonaMautrix.swift

```diff
@@ -21,10 +21,18 @@ private let trace = Tracer(log, true)
 class BarcelonaMautrix {
     static let shared = BarcelonaMautrix()
     
-    let reader = BLPayloadReader()
+    private let mautrixIPCChannel: MautrixIPCChannel
+    private let reader: BLPayloadReader
+    private let eventHandler: BLEventHandler
+    
+    init() {
+        mautrixIPCChannel = MautrixIPCChannel(inputHandle: FileHandle.standardInput, outputHandle: FileHandle.standardOutput)
```

Comment:
Wait, won't this still write to stdout? Shouldn't we change this to something different?